transformers/integration_tests/cat_file_test.go

```diff
@@ -266,6 +266,48 @@ var _ = Describe("Cat File transformer", func() {
 			Expect(dbResult.What).To(Equal("box"))
 			Expect(dbResult.Data).To(Equal("30000000000000000000000000000000000000000000000000000"))
 		})
+
+		It("persists a chop lump dunk event (dunk)", func() {
+			chopDunkBlockNumber := int64(10769102)
+			header, err := persistHeader(db, chopDunkBlockNumber, blockChain)
+			Expect(err).NotTo(HaveOccurred())
+			catFileConfig.TransformerName = constants.CatFileChopLumpDunkTable
+			catFileConfig.Topic = constants.CatFileChopLumpDunkSignature()
+			catFileConfig.StartingBlockNumber = chopDunkBlockNumber
+			catFileConfig.EndingBlockNumber = chopDunkBlockNumber
+
+			initializer := event.ConfiguredTransformer{
+				Config:      catFileConfig,
+				Transformer: chop_lump_dunk.Transformer{},
+			}
+			transformer := initializer.NewTransformer(db)
+
+			logs, err := logFetcher.FetchLogs(
+				[]common.Address{common.HexToAddress(catFileConfig.ContractAddresses[0])},
+				[]common.Hash{common.HexToHash(catFileConfig.Topic)},
+				header)
+			Expect(err).NotTo(HaveOccurred())
+
+			eventLogs := test_data.CreateLogs(header.Id, logs, db)
+
+			err = transformer.Execute(eventLogs)
+			Expect(err).NotTo(HaveOccurred())
+
+			var dbResults []catFileChopLumpDunkModel
+			msgSenderID, err := shared.GetOrCreateAddress("0xBE8E3e3618f7474F8cB1d074A26afFef007E98FB", db)
+			Expect(err).NotTo(HaveOccurred())
+			err = db.Select(&dbResults, `SELECT address_id, msg_sender, what, data FROM maker.cat_file_chop_lump_dunk`)
+			Expect(err).NotTo(HaveOccurred())
+			addressID, err := shared.GetOrCreateAddress(test_data.Cat110Address(), db)
+			Expect(err).NotTo(HaveOccurred())
+
+			Expect(dbResults[0].AddressID).To(Equal(addressID))
+			Expect(dbResults[0].MsgSender).To(Equal(msgSenderID))
+			Expect(dbResults[0].What).Should(Or(Equal("dunk"), Equal("chop")))
```

Comment:
Sorry this came up earlier and I think I provided a bad answer, but I think we might want to do a sort here - similarly to what we do for [`vat_move`](https://github.com/makerdao/vdb-mcd-transformers/blob/staging/transformers/integration_tests/vat_move_test.go#L72). Probably not blocking but would enable us to be sure we were asserting against a result where `What` == `dunk` (and specify a single `data` value)